config/initializers/airbrake.rb

```diff
@@ -1,7 +1,17 @@
-if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
-  require 'airbrake'
+require 'airbrake'
 
+if !defined?(Rails) || Rails.env.staging? || Rails.env.production?
   Airbrake.configure do |config|
     config.api_key = ENV['AIRBRAKE_API_KEY']
   end
+else
+  module Airbrake
```

Comment:
how about a `Samson.exception_handler` callback that gets a message +
exception ?

On Wed, Aug 19, 2015 at 4:32 PM, Gabe Martin-Dempesy <
notifications@github.com> wrote:

> In config/initializers/airbrake.rb
> <https://github.com/zendesk/samson/pull/535#discussion_r37481504>:
>
> >    Airbrake.configure do |config|
> >      config.api_key = ENV['AIRBRAKE_API_KEY']
> >    end
> > +else
> > +  module Airbrake
>
> The ideal situation here would be to use an exception notification service
> adapter. I'm surprised there isn't any public gem (that I found) that
> handles this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendesk/samson/pull/535/files#r37481504>.
>